internal/flags/flags.go

```diff
@@ -240,6 +240,12 @@ Should only be used for testing.`)
 		viper.GetString("WATCHTOWER_NOTIFICATION_EMAIL_SUBJECTTAG"),
 		"Subject prefix tag for notifications via mail")
 
+	flags.StringP(
+		"notification-email-subjecthostname",
+		"",
+		viper.GetString("WATCHTOWER_NOTIFICATION_EMAIL_SUBJECTHOSTNAME"),
+		"Subject custom hostname for notifications via mail")
+
```

Comment:
```suggestion
	flags.StringP(
		"notification-hostname",
		"",
		viper.GetString("WATCHTOWER_NOTIFICATION_HOSTNAME"),
		"Custom hostname for notification titles")
```

Reply:
Adjusted in https://github.com/containrrr/watchtower/pull/994/commits/b3bbc3ca0c676c04e700aff5e337cd5ab005b48a.

Also added the ability to override the hostname for all notifiers.